snapraid-aio-script.sh

```diff
@@ -286,20 +284,19 @@ function sanity_check() {
 
   echo "Testing that all parity files are present."
   mklog "INFO: Testing that all parity files are present."
-  for i in "${PARITY_FILES[@]}"
-    do
-      if [ ! -e "$i" ]; then
-        echo "[$(date)] ERROR - Parity file ($i) not found!"
-        echo "ERROR - Parity file ($i) not found!" >> "$TMP_OUTPUT"
-    echo "**ERROR**: Please check the status of your disks! The script exits here due to missing file or disk..."
-    mklog "WARN: Parity file ($i) not found!"
-    mklog "WARN: Please check the status of your disks! The script exits here due to missing file or disk..."
+  for i in "${PARITY_FILES[@]}"; do
+    if [ ! -e "$i" ]; then
```

Comment:
This bit is unable to parse multiple parity files. 
Maybe this is an unusual SnapRAID configuration, but supported by this script, and I use it for testing purposes.
With this code, I get an error like:

```
Testing that all parity files are present.
ERROR - Parity file (/srv/dev-disk-by-label-DISK3/snapraid.parity
/srv/dev-disk-by-label-DISK4/snapraid.2-parity) not found!
```
These are two files, but seems like are being parsed like only one, because the error is printed only once.
Files are there, in fact if remove double quotes like below, the script is able to parse the files and proceed.

`if [ ! -e $i ]; then`

Please tell me what you think.

Reply:
Strange, I ran a basic test and it looks correct:

```
ARRAY=(1 2 3 4 5)
for i in ${ARRAY[@]}; do
  echo "item $i"
done
```

output:
```
item 1
item 2
item 3
item 4
item 5
```